js/.eslintrc.js

```diff
@@ -0,0 +1,232 @@
+module.exports = {
```

Comment:
Our eslint config should enforce 2 space indents in js/ts files. Probably should also itself use 2 space indents as well

Reply:
updated